<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-8CD7N2K" name="com.rameshsoft.automation.seleniumcore.DataProviderTest" tests="1" failures="0" timestamp="9 Mar 2019 03:04:57 GMT" time="0.025" errors="1">
  <testcase name="login" time="0.025" classname="com.rameshsoft.automation.seleniumcore.DataProviderTest">
    <error type="org.testng.TestNGException" message="
The data provider is trying to pass 3 parameters but the method com.rameshsoft.automation.seleniumcore.DataProviderTest#login takes 2">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 3 parameters but the method com.rameshsoft.automation.seleniumcore.DataProviderTest#login takes 2
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1225)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1118)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- login -->
</testsuite> <!-- com.rameshsoft.automation.seleniumcore.DataProviderTest -->
